silverback/application.py

```diff
@@ -88,6 +107,92 @@ def __init__(self, settings: Settings | None = None):
             f"{signer_str}{start_block_str}{new_block_timeout_str}"
         )
 
+        # NOTE: The runner needs to know the set of things that the app is tracking as a parameter
+        # NOTE: We also need to know the defaults in case the parameters are not in the backup
+        self.parameter_defaults: dict[str, ScalarType | None] = dict()
+
+        self._create_system_startup_task()
+        self._create_system_backup_task()
+
+    def _create_system_startup_task(self):
+        # Add a task to load all parameters from state at startup
+        async def startup_handler() -> StateSnapshot:
+            # NOTE: This is not networked in any way, nor thread-safe nor multi-process safe,
+            #       but will be accessible across multiple workers in a single container
+            # NOTE: *DO NOT USE* in Runner, as it will not be updated by the app
+            self.state = SharedState()
+            # NOTE: attribute does not exist before this task is executed,
+            #       ensuring no one uses it during worker startup
+
+            self.backup = AppDatastore()
```

Comment:
thought: I think there's some confusion happening here.  This is worker-level stuff, but it's handling `last_block_*` values where are runner-controlled data (`self.state` on the runner).  So now each worker is also tracking it, but is it ever updated?  Could each worker drift from each-other?  

This may appear to work on single-process silverback, but would not on a distributed system.

Reply:
> thought: I think there's some confusion happening here. This is worker-level stuff, but it's handling `last_block_*` values where are runner-controlled data (`self.state` on the runner). So now each worker is also tracking it, but is it ever updated? Could each worker drift from each-other?
> 
> This may appear to work on single-process silverback, but would not on a distributed system.

For a single container app running with multiple workers, they all share the same state, and all the tasks are only handled by one worker. I tested this mechanism works using 10 workers in my worker process (incrementing a shared state with every task did not lead to out of order execution for a dozens of logs at once at least)

This may eventually need some sort of concurrency lock, but the basic mechanism works